### DISCUSSION TITLE
Is there any solution to read Polish characters from file?

### DISCUSSION THREAD

**Discussion by sndth** [nlohmann/json#3913] [Q&A]
Hello, how can I read from JSON file Polish characters? Is it even possible?
I mean values, for example:
```
{
  "Key": "Żółć"
}
```

**Comment by nlohmann** [comment]
Yes: https://json.nlohmann.me/home/faq/#parse-errors-reading-non-ascii-characters

**Reply by sndth** [reply]
For me code from example (with to_utf8 function) not work.
![123](https://user-images.githubusercontent.com/75499293/211212985-db1d114f-9885-4829-8de9-37afd28db02e.png)

**Reply by nlohmann** [reply]
Are you reading from std::wstring? These are not supported. Please use an std::string.

**Reply by sndth** [reply]
I used code from example for test:
![test](https://user-images.githubusercontent.com/75499293/211214249-b3f61eda-f191-475c-bfbb-438a90868a2d.png)
But my output is completely other than result on page.

**Reply by nlohmann** [reply]
Then your code is somewhere using a different encoding.

**Reply by sndth** [reply]
So how can I change encoding? My CPP file has UTF8.

**Reply by nlohmann** [reply]
I don't know how this works in MSVC.
